packages/control-property-editor/test/unit/App.test.tsx

```diff
@@ -182,6 +182,42 @@ test('renders warning dialog for "FE_FROM_SCRATCH" scenario', async () => {
     fireEvent.click(okButton);
 });
 
+test('renders warning message for "ADAPTATION_PROJECT" scenario', async () => {
+    const initialState = {
+        deviceType: DeviceType.Desktop,
+        scale: 1.0,
+        selectedControl: undefined,
+        outline: [],
+        filterQuery: [
+            { name: FilterName.focusEditable, value: true },
+            { name: FilterName.focusCommonlyUsed, value: true },
+            { name: FilterName.query, value: '' },
+            { name: FilterName.changeSummaryFilterQuery, value: '' },
+            { name: FilterName.showEditableProperties, value: true }
+        ],
+        scenario: scenario.UiAdaptation,
+        isAdpProject: true,
+        icons: [],
+        changes: {
+            controls: {},
+            pending: [],
+            saved: [],
+            pendingChangeIds: []
+        },
+        dialogMessage: {
+            message: 'Some Text',
+            shouldHideIframe: false
+        }
+    };
+    render(<App previewUrl="" scenario="ADAPTATION_PROJECT" />, { initialState });
+
+    const warningDialog = screen.getByText(/Some Text/i);
+    expect(warningDialog).toBeInTheDocument();
+    const okButton = screen.getByText(/ok/i);
+    expect(okButton).toBeInTheDocument();
+    fireEvent.click(okButton);
```

Comment:
After clicking the OK button maybe we can test that the dialog has closed, otherwise, this line is redundant.

Reply:
Done